wrappers/csharp/Intel.RealSense/Devices/AutoCalibratedDevice.cs

```diff
@@ -11,7 +11,7 @@ public class AutoCalibratedDevice : CalibratedDevice
         internal AutoCalibratedDevice(IntPtr dev)
             : base(dev)
         { }
-        public static AutoCalibratedDevice FromDevice(Device dev)
+        public static new AutoCalibratedDevice FromDevice(Device dev)
```

Comment:
`new` in a function signature?

Reply:
Yes, apparently, we use it in C# when we 'hide' other functions, here this FromDevice function is hiding its base class's FromDevice function
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/new-modifier
https://stackoverflow.com/a/1014378